x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/use_upgrade_prebuilt_rules_table_columns.tsx

```diff
@@ -142,8 +143,48 @@ const MODIFIED_COLUMN: TableColumn = {
   truncateText: true,
 };
 
+const CONFLICT_COLUMN: TableColumn = {
+  field: 'diff.field_conflict_level',
+  name: <RulesTableEmptyColumnName name={i18n.COLUMN_CONFLICT} />,
+  align: 'center',
+  render: (conflictLevel: ThreeWayDiffConflict) => {
+    if (conflictLevel === ThreeWayDiffConflict.NONE) {
+      return null;
+    }
+
+    if (conflictLevel === ThreeWayDiffConflict.SOLVABLE) {
```

Comment:
Do we want this to be based on the internal state where we calculate the field conflicts (e.g. `hasFieldConflicts`, `hasNonSolvableFieldConflicts`) so it disappears when conflicts are addressed? Or do you think that would be weird when combined with the update all request where we don't use the internal resolved values? cc @maximpn

Reply:
When have different ways users can use to bulk upgrade rules

- `Upgrade All` which doesn't accept any rule specifiers. It means if a user resolved rule upgrade conflicts in rule upgrade flyout it's still impossible to bulk upgrade that rule via `Upgrade All` option.
- `Upgrade Selected` allows to bulk upgrade selected rules + rules with resolved conflicts via rule upgrade flyout.

I thinks we should stick to the original state (which doesn't take into account whether conflicts resolved in the prebuilt rule upgrade flyout) for consistency.